cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/CassandraCradleStorage.java

```diff
@@ -86,7 +86,7 @@ public CassandraCradleStorage(CassandraConnectionSettings connectionSettings, Ca
 			ExecutorService composingService) throws CradleStorageException
 	{
 		super(composingService, storageSettings.getMaxMessageBatchSize(), storageSettings.getMaxTestEventBatchSize());
-		this.connection = new CassandraConnection(connectionSettings);
+		this.connection = new CassandraConnection(connectionSettings, storageSettings);
```

Comment:
@MrJonniK `CassandraStorageSettings` is related to Cradle and how the data storage is organized. `CassandraConnection` is related to connectivity with Cassandra, not with data storage. So it should not require `CassandraStorageSettings`. As `CassandraConnection` needs timeout setting, pass timeout to `CassandraConnection` explicitly. Or even move timeout from `CassandraStorageSettings` to `CassandraConnectionSettings`.